daemon/cluster/noderunner.go

```diff
@@ -52,6 +53,10 @@ type nodeStartConfig struct {
 	AdvertiseAddr string
 	// DataPathAddr is the address that has to be used for the data path
 	DataPathAddr string
+	// DefaultAddressPool contains list of subnets
+	DefaultAddressPool []*net.IPNet
+	// SubnetSize contains subnet size of DefaultAddressPool
+	SubnetSize int
```

Comment:
Same question here (`int` vs `uint32`)

Reply:
modified it. But SwarmKit still has int. only we will have to convert at present before calling swarmkit API.